jupyter_client/kernelspec.py

```diff
@@ -136,8 +122,8 @@ def _get_destination_dir(self, kernel_name, user=False):
         if user:
             return os.path.join(self.user_kernel_dir, kernel_name)
         else:
-            if SYSTEM_KERNEL_DIRS:
-                return os.path.join(SYSTEM_KERNEL_DIRS[-1], kernel_name)
+            if SYSTEM_JUPYTER_PATH:
+                return os.path.join(SYSTEM_JUPYTER_PATH[0], 'kernels', kernel_name)
             else:
                 raise EnvironmentError("No system kernel directory is available")
```

Comment:
This case shouldn't be necessary now, right? We're ensuring that there's always at least 1 path in `SYSTEM_JUPYTER_PATH`.